src/main/java/com/sigopt/model/TrainingEarlyStoppingCriteria.java

```diff
@@ -0,0 +1,20 @@
+package com.sigopt.model;
+
+public class TrainingEarlyStoppingCriteria extends StructObject {
+    public TrainingEarlyStoppingCriteria() {
+        super();
+    }
+
+    public String getName() {
+        return (String) this.get("name");
+    }
+
+    public String getMetric() {
+        return (String) this.get("metric");
+    }
+
+    public String getType() {
+        return (String) this.get("type");
+    }
+    
```

Comment:
needs Builder

Reply:
Having the builder here messed up the inherited classes, moving making them all inherit from API directly so this is gone anyways.